examples/app.R

```diff
@@ -232,21 +232,21 @@ tabs <- function () {
     )
   )
 }
-uilist <- function() {
+uilist_demo <- function() {
+  list_content <- data.frame(
+    header = paste("Header", 1:5),
+    description = paste("Description", 1:5),
+    stringsAsFactors = FALSE
+  )
+
   div(
     h1(class="ui dividing header", id="list", "List"),
-    demo(div(class="ui list", c("Apples", "Pears", "Oranges") %>% purrr::map(~ div(class="item", .)))),
-    demo(div(class="ui list", c("Apples", "Pears", "Oranges") %>% purrr::map(~
-      div(class="item", uiicon('users'), div(class="content", .))))),
-    demo(div(class="ui relaxed divided list", c("Apples", "Pears", "Oranges") %>% purrr::map(~
-      div(class="item",
-          uiicon('large github middle aligned'),
-          div(class="content",
-              a(class="header", "Hello"),
-              div(class="description", .))
-      ))))
+    demo(uilist(list_content, "", is_divided = FALSE, is_description = FALSE)),
```

Comment:
What about situation when user wants to assign different icon to each element of the list. I think we should support this as well, WDYT?

Reply:
:+1: I like the idea. Can we do this outside this PR and create an issue for this?

Reply:
:+1: I like the idea. Can we do this outside this PR and create an issue for this? So this is not blocking this one. You can assign me this new PR